### PR TITLE
jormungandr: ptref test on added trip objects added

### PR DESCRIPTION
**Pull request by kadhikari**
https://jira.kisio.org/browse/NAVP-1270

**To be added:** same block of test after updating the trip recently added

### CODE REVIEW

**Comment by kadhikari** [comment]
jenkins, test this please

File: source/jormungandr/tests/kirin_realtime_tests.py [hunk 1]
```diff
@@ -1757,6 +1757,194 @@ def test_add_new_trip(self):
...
+        resp = self.query_region("lines/line:stopC_stopB/networks")
+        assert resp["networks"][0]["id"] == "network:additional_service"
+
+        # The physical_mode sent in gtfs-rt should be present in the new line added
+        resp = self.query_region("lines/line:stopC_stopB/physical_modes")
+        assert resp["physical_modes"][0]["id"] == "physical_mode:Bus"
+
+        # The default commercial_mode used for a new line should be present
+        resp = self.query_region("lines/line:stopC_stopB/commercial_modes")
+        assert resp["commercial_modes"][0]["id"] == "commercial_mode:additional_service"
```
Comment (nonifier) [inline source/jormungandr/tests/kirin_realtime_tests.py:1842]: I'm maybe wrong, but shouldn't we be able to set the commercial mode when sending the disruption ? 
Do we always attach all new trips to this default `"commercial_mode:additional_service"`mode ?

File: source/jormungandr/tests/kirin_realtime_tests.py [hunk 2]
```diff
@@ -1757,6 +1757,194 @@ def test_add_new_trip(self):
...
+        # The following ptref search should work with theorical data.
+        # network <-> datasets
+        resp = self.query_region("networks/base_network/datasets")
+        assert resp["datasets"][0]["id"] == "default:dataset"
+        resp = self.query_region("datasets/default:dataset/networks")
+        assert resp["networks"][0]["id"] == "base_network"
+
+        # line <-> company
+        resp, status = self.query_region("lines/A/companies", check=False)
+        assert status == 404
```
Comment (nonifier) [inline source/jormungandr/tests/kirin_realtime_tests.py:1869]: Does this means that we need to fix the links with a 404 ?

**Review by nonifier — APPROVED** [review]
Very interesting test. 
Looks like there is a lot of work on `pt_ref` that needs to be done :/

**Review by benoit-bst — APPROVED** [review]
It is truly comprehensive. 
You are a Kirin test Jedi master

**Pull request merged**